RELEASE.rst

```diff
@@ -0,0 +1,9 @@
+RELEASE_TYPE: patch
+
+This release improves the performance of
+:func:`hypothesis.strategies.characters` when using ``blacklist_characters``
```

Comment:
Style nit: de-qualify references to strategies with a tilde after the first backtick.